app/views/pattern_library/index.slim

```diff
@@ -32,7 +34,7 @@ h2 Molecules
 
 h2 Organisms
 
-== pl_component 'ContentSection', { title: 'Title', subtitle: 'Some description', label: 'Text input', id: 'text-area', name: 'Area', placeholder: 'Enter text', describeId: 'lable', note: 'More text', error: 'Please enter a First Name' }
+/== pl_component 'ContentSection', { title: 'Title', subtitle: 'Some description', label: 'Text input', id: 'text-area', name: 'Area', placeholder: 'Enter text', describeId: 'lable', note: 'More text', error: 'Please enter a First Name' }
```

Comment:
any reason why we're keeping the commented out pl_components?

Reply:
no reason, removing